packages/component-base/src/polylit-mixin.js

```diff
@@ -105,7 +106,7 @@ const PolylitMixinImplementation = (superclass) => {
             this.requestUpdate(name, oldValue, options);
 
             // Enforce synchronous update
-            if (this.hasUpdated) {
```

Comment:
Please note that there's a separate block for readOnly properties (line 126) - that needs to be updated as well. Also we need to have separate tests for readOnly and normal properties.

Reply:
Yeah, thanks. 

Unfortunately, I'm going to put this fix on hold because it causes various test failures, and I'm not sure what the reason is.